src/OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -48,6 +49,14 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
 
 #define ERROR(format, ...) LOG("Error: " format, ##__VA_ARGS__)
 
+#define LOG_ONCE(format, ...)                                                  \
+  do {                                                                         \
+    static std::once_flag already_logged_flag;                                 \
+    std::call_once(already_logged_flag, []() { LOG(format, ##__VA_ARGS__); }); \
```

Comment:
The lambda might need `&` as a capture argument

Reply:
And could you add some usage of these macros somewhere? Otherwise they won't be tested.

Reply:
I tried it manually (as I was intending to use it in a different PR #2051). It worked without the `&`, but I can safely add it. I also added a usage.

Reply:
Yes, it worked because `format` is a constexpr expression, and __VA_ARGS__ is empty in your example. Have you tried it with an argument and it still worked? I think it would break when the ampersand is not there and the macro gets an argument.

Anyways, thanks a lot for addressing the comment. LGTM!

Reply:
You are totally right, tested it locally, and it did not compile.